bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml

```diff
@@ -48,6 +48,11 @@ limitations under the License.
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
```

Comment:
I think you need to an entry for the other slf4j dependency here as well

Reply:
good catch

Reply:
actually, because we changed the version to 1.7.25, this is no longer an issue, but can add for consistency